code/game/machinery/cloning.dm

```diff
@@ -364,12 +378,12 @@
 		use_power(200)
 
 //Let's unlock this early I guess.  Might be too early, needs tweaking.
-/obj/machinery/clonepod/attackby(obj/item/I, mob/user, params)
-	if(exchange_parts(user, I))
+/obj/machinery/clonepod/attackby(obj/item/item, mob/user, params)
```

Comment:
No need to rename `I` to `item`. `I` is more commonly used. Usually people call scoped variables in this fashion.

Reply:
Okay, thanks. I don't know a lot of convention, thanks for showing me.